dynamo/configuration.py

```diff
@@ -137,6 +140,35 @@ def allowed_X_layer_names():
     def init_uns_pp_namespace(adata: AnnData):
         adata.uns[DynamoAdataKeyManager.UNS_PP_KEY] = {}
 
+    def get_excluded_layers(
+        X_total_layers: bool = False,
+        splicing_total_layers: bool = False
+    ):
+        """ Get a list of excluded layers based on the provided arguments."""
+        excluded_layers = []
+        if not X_total_layers:
+            excluded_layers.extend(["X"])
+        if not splicing_total_layers:
+            excluded_layers.extend(["spliced", "unspliced"])
+        return excluded_layers
+
+    def aggregate_layers_into_total(
+        _adata: AnnData,
+        layers: Union[str, List[str]] = "all",
+        total_layers: Union[List[str], None] = None,
+        extend_layers: bool = True,
+    ):
+        """Create a total layer in adata by aggregating multiple layers."""
+        if not isinstance(total_layers, list):
+            total_layers = [total_layers]
+        if len(set(total_layers).difference(_adata.layers.keys())) == 0:
+            total = None
+            for t_key in total_layers:
+                total = _adata.layers[t_key] if total is None else total + _adata.layers[t_key]
+            _adata.layers["_total_"] = total
+            if extend_layers:
+                layers.extend(["_total_"])
+        return total_layers, layers
 
```

Comment:
this function is not clear to me. Can you also document each argument? 
Are layers are valid layers in the adata while total_layers are the layers that should be used to calculate total RNA for collapsing?

Reply:
Added a docstring for this helper function. The size factor normalization function is able to calculate size factors from customized layers. Given a list of total_layers, this helper function will calculate a temporary `_total_` layer. Layers are the list of layers users give in the size_factor normalization function.